src/app.js

```diff
@@ -1 +1,49 @@
-// write code here
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const moveFile = (targetFile, moveTo) => {
+  try {
+    if (targetFile === moveTo) {
+      console.log('Initial file and target destination are the same');
+
+      return;
+    }
+
+    let targetIsDirectory = false;
+
+    if (fs.existsSync(moveTo)) {
+      targetIsDirectory = fs.lstatSync(moveTo).isDirectory();
+    } else if (moveTo.endsWith('/')) {
+      console.error('Error: Destination directory does not exist');
```

Comment:
Consider using `console.log` instead of `console.error` for user feedback messages that are not actual errors, like informing the user that a directory does not exist.